src/cpl/share_esmf/laiStreamMod.F90

```diff
@@ -144,7 +146,7 @@ subroutine lai_init(bounds)
          stream_yearAlign    = model_year_align_lai,             &
```

Comment:
Above this there needs to be a shr_mpi_bcast call for lai_dtlimit as well (and it looks like this is missing for lai_mapalgo). I also suggest adding to the logging write statements as well (and same for lai_mapalgo).

Reply:
done

Reply:
@ekluzek I think I've done this correctly, notably the critical part to "add shr_mpi_bcast call for lai_dtlimit.", but can you please confirm?

If this look OK, I think we can merge this PR to @olyson's PR and maybe @slevis-lmwg work to bring this to b4b-dev?  Having more functional usermods would be helpful for @TeaganKing to make progress on the larger PLUMBER project.

Reply:
Just one change here, lai_dtlimit is a real rather than character, so remove the trim() function call around it.

Reply:
Everything else looks great. So I'll mark as good to go...